src/DependencyInjection/CmfRoutingAutoExtension.php

```diff
@@ -65,6 +67,14 @@ public function load(array $configs, ContainerBuilder $container)
                 $adapterName = 'doctrine_phpcr_odm';
             }
             $container->setParameter('cmf_routing_auto.persistence.phpcr.route_basepath', $config['persistence']['phpcr']['route_basepath']);
+
+            $commandService = $container->getDefinition('cmf_routing_auto.auto_route_refresh_command');
+            $commandService->replaceArgument(2, new Reference('doctrine_phpcr.odm.default_document_manager'));
+            $commandService->replaceArgument(3, new Reference('cmf_routing_auto.adapter.phpcr_odm'));
+        }
```

Comment:
It might make more sense to have a `RefreshRoutes` service which is passed to the command. This service could then be invoked from other contexts (e.g. a web UI). You could then create an implementation for either the ORM or the PHPCR-ODM, and you will not be limited to the dependencies accepted by the command.

For logging you could pass a closure.